lambda/src/main/scala/pricemigrationengine/model/BillingPeriod.scala

```diff
@@ -0,0 +1,26 @@
+package pricemigrationengine.model
+
+object BillingPeriod extends Enumeration {
+  type BillingPeriod = Value
+  val Month, Quarterly, Annual = Value
```

Comment:
I thought there were model billing periods than this, like semiannual, unless they have all been normalised, which would make things easier

Reply:
Fixed here: https://github.com/guardian/price-migration-engine/pull/941